vocabularies/Hierarchy.xml

```diff
@@ -37,35 +37,320 @@
+which are specific for the given annotation qualifier.
+These actions MUST have the same signature as the template actions linked below, with
+`Edm.EntityType` replaced with the entity type on which the recursive hierarchy is defined.
+The resource path of the binding parameter MUST traverse the hierarchy collection,
+including any hierarchy directory.
+
+The template actions themselves cannot be invoked.</String>
+        </Annotation>
+        <Property Name="ChangeNextSiblingAction" Type="Common.QualifiedName" Nullable="true">
+          <Annotation Term="Core.Description" String="Action that moves a node among its siblings, following [this template](#Template_ChangeNextSiblingAction)" />
         </Property>
-        <Property Name="SiblingRankProperty" Type="Edm.PropertyPath" Nullable="true">
-          <Annotation Term="Core.Description" String="Property holding the sibling rank of a node" />
-          <Annotation Term="Core.LongDescription" String="The sibling rank of a node is the index of the node in the sequence of all nodes with the same parent created by preorder traversal of the hierarchy structure after evaluating the $filter expression in the request excluding any conditions on key properties. The first sibling is at position 0." />
+        <!--
+        <Property Name="ChangePreviousSiblingAction" Type="Common.QualifiedName" Nullable="true">
+          <Annotation Term="Core.Description" String="Action that moves a node among its siblings, following [this template](#Template_ChangePreviousSiblingAction)" />
+        </Property>
+        -->
+        <Property Name="CopyAction" Type="Common.QualifiedName" Nullable="true">
+          <Annotation Term="Core.Description" String="Action that copies a node and its descendants, following [this template](#Template_CopyAction)" />
+        </Property>
+      </ComplexType>
+
+      <Term Name="MatchCount" Type="Edm.Int64" Nullable="false" AppliesTo="Collection">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Common.IsInstanceAnnotation" />
+        <Annotation Term="Core.Description" String="Instance annotation on the result of an `$apply` query option containing the number of matching nodes after hierarchical transformations" />
+        <Annotation Term="Core.LongDescription">
+          <String>The service MAY designate a subset of the `$apply` result as "matching nodes".
+          For requests following the pattern described [here](#RecursiveHierarchyType), this subset is the output set of the
+          `filter` or `search` transformation that occurs as the fourth parameter
+          of the last `ancestors` transformation or occurs nested into it.&lt;br&gt;
+          For requests not following this pattern, the subset NEED NOT be defined.&lt;br&gt;
+          This instance annotation is available if [`RecursiveHierarchy/MatchedProperty`](#RecursiveHierarchyType)
```

Comment:
I don't see a `MatchedProperty`, only `Matched` and `MatchedDescendantCountPropert`.